src/pages/SitePolicy/SitePolicy.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import { Accordion, Card, Container } from 'react-bootstrap';
+import styles from './SitePolicy.module.scss';
+import {PrivacyPolicy} from './policies/PrivacyPolicy';
+import {TermsOfUse} from './policies/TermsOfUse';
+import {Disclaimer} from './policies/Disclaimer';
+import {CookiePolicy} from './policies/CookiePolicy';
+import {DisputeResolution} from './policies/DisputeResolution';
+
+export default class SitePolicy extends React.Component {
+  render() {
+    return (
+      <React.Fragment >
+          <Container>
+          <h1 className={styles['headline']}>Site Policy</h1>
+          <h5 className={styles['update']}>Last Updated: &nbsp; <i> January 6th, 2020 </i></h5>
+          <Accordion>
+            <Card>
+              <Accordion.Toggle as={Card.Header} eventKey='termsOfUse' >
```

Comment:
The headers don't provide feedback that you can press them, I would add a class named `header` in the stylesheet with  `cursor: pointer;` as the style and add it to all the `Accordion.Toggle` elements in this file.